source/eitri/eitri.py

```diff
@@ -124,10 +124,7 @@ def get_params():
         help="the output file path (default: './data.nav.lz4')",
     )
     parser.add_argument(
-        '-i',
-        '--cities-file',
-        type=argparse.FileType,
-        help='the path to the "cities" file to load  (usually a *.osm.pbf)',
+        '-i', '--cities-file', type=str, help='the path to the "cities" file to load  (usually a *.osm.pbf)'
```

Comment:
Maybe as above:
```suggestion
        '-i', '--cities-file', type=is_valid_file, help='the path to the "cities" file to load  (usually a *.osm.pbf)'
```

Reply:
:negative_squared_cross_mark: It won't work as the file exists (of course) and:
https://github.com/CanalTP/navitia/blob/3cdfe8bf9156264cc04b93056b565df08fc52c29/source/eitri/eitri.py#L71

Reply:
OK, we may have the same kind of test for existing files, but it's already overkill I think :+1: